scripts/alerts-during-perf-testing.sh

```diff
@@ -8,7 +8,7 @@
 # - jq
 # - oc (logged in at the cmd line in order to get the bearer token)
 # VARIABLES
-NAMESPACE_PREFIX=redhat-rhmi-
+NAMESPACE_PREFIX=redhat-rhoam-
```

Comment:
I think we can get the namespace prefix dynamically, e.g:
https://github.com/integr8ly/integreatly-operator/blob/master/scripts/setup-sso-idp.sh#L8

But given we do perf testing only against RHOAM I think it is optional

Reply:
:+1: